client/src/App.js

```diff
@@ -1,18 +1,14 @@
 import './App.css';
+import ExamDetails from './ExamDetails';
 
-import { useApi } from './hooks/use-api';
+import { ChakraProvider } from '@chakra-ui/react'
 
 function App() {
-  const { response } = useApi();
 
   return (
-    <div className="App">
-      <header className="App-header">
-        <p>
-          {response}
-        </p>
-      </header>
-    </div>
+    <ChakraProvider>
```

Comment:
While this is great to be provided, this isn't enough for a PR just yet
Please add the respective pages pertaining to issues:
#24
#26
and then a PR can be requested for merging